.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
I really don't like pre-commit hooks that depend on anything that is not completely standard. Can we please not have them?

I don't care if you set it up for yourself, but I don't want it in the repository for myself.

Reply:
I'll remove this and set up CI